frontend/public/components/monitoring/dashboards/custom-time-range-modal.tsx

```diff
@@ -56,8 +66,7 @@ const CustomTimeRangeModal = ({ cancel, close, activePerspective }: CustomTimeRa
     endTime ? toISOTimeString(new Date(endTime)) : '23:59',
   );
 
-  const submit: React.FormEventHandler<HTMLFormElement> = (e) => {
-    e.preventDefault();
+  const submit: React.FormEventHandler<HTMLButtonElement> = () => {
     const from = Date.parse(`${fromDate} ${fromTime}`);
     const to = Date.parse(`${toDate} ${toTime}`);
     if (_.isInteger(from) && _.isInteger(to)) {
```

Comment:
Should we check that the `from` date is before `to`?

Reply:
Yes, I think it should!

I'd like to leave that for a separate PR since I'm intending for this PR to just be refactoring.

Reply:
Bug ticket for this: https://issues.redhat.com/browse/OU-131